api-schema/xdb.yaml

```diff
@@ -328,6 +324,26 @@ components:
         - SCHEDULED_TIMER
         - FIRED_TIMER
         - SKIPPED_TIMER
+    LocalQueueResult:
+      type: object
+      required:
+        - status
+        - queueName
+      properties:
+        status:
+          $ref: '#/components/schemas/LocalQueueStatus'
+        queueName:
+          type: string
+        messages:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocalQueueMessageResult'
+    LocalQueueStatus:
```

Comment:
probably `LocalQueueCommandStatus` or just `QueueCommandStatus` is more accurate. QueueStatus could mean something else for representing the status of the whole queue. 

I am also open to combine this with TimerStatus to `CommandStatus`:
* WAITING_COMMAND
* COMPLETED_COMMAND
* SKIPPED_COMMAND

Reply:
Combining is a good idea to me.